runtime/src/bank.rs

```diff
@@ -635,21 +636,34 @@ impl Bank {
     where
         F: Fn(&VoteState) -> bool,
     {
-        self.accounts
```

Comment:
This was a fine function in its original form. No reason to crank up the cyclomatic complexity like this. Post-merge, can you put it back, rename it, make it private, and make a new pub `vote_states()` that builds upon the old one.

Reply:
Nevermind, not that important. I see the `exists` trick you added. You could layer as I suggested, but meh.